molecule/default/verify.yml

```
@@ -13,3 +14,60 @@
     - name: Check for backup
       stat:
         path: /opt/backup/rsnapshot/hourly.0/backup-defaul/root/rsnapshot/timestamp.hourly
+
+    - name: .my.cnf exists    
+      command: cat /root/.my.cnf
+
+    - name: Create test table in database and insert test value
+      community.mysql.mysql_query:
+        query:
+          - CREATE TABLE IF NOT EXISTS test (message varchar(255) NOT NULL) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+          - INSERT INTO test(message) VALUES('Ansible value');
+        login_db: testDatabase
+
+    - name: Create backup script
+      template:
+        src: /home/runner/work/ansible-collection-backup/ansible-collection-backup/roles/backup/templates/backup.sh.j2
```

```suggestion
        src: backup.sh.j2
```
I guess, `template` can use relative names and search for files in `templates`.

---

Scratch that, it looks like you don't need that task anyway.